app/views/products/index.html.erb

```diff
@@ -17,7 +17,7 @@
         <% end %>
         <td><%= product.name %></td>
         <td><%= product.description %></td>
-        <td><%= number_to_currency product.unit_price %></td>
+        <td><%= number_to_currency product&.unit_price_in_cents %></td>
```

Comment:
A lonely operator doesn't work here since the method wont be empty right?  I mean it literally doesn't work for me anyways because it still tries to call the method but then can't divide without a price